addon/controllers/new.js

```diff
@@ -6,7 +6,7 @@ export default Ember.Controller.extend({
     post (title, body) {
       let blog = this.get('blog')
       blog.add(title, body)
-      this.transitionToRoute('/blog')
+      this.transitionToRoute('index')
```

Comment:
I'm getting the following error: Route#transitionTo cannot be used for URLs. Please use the route name instead.